Packs/GCP/Integrations/GCP/GCP_test.py

```diff
@@ -2469,6 +2469,393 @@ def test_gcp_compute_instance_label_set_command_add_labels_true(mocker):
     assert result.outputs == mock_operation_response
 
 
+def test_validate_bucket_policy_for_set_add_mode_valid():
+    """
+    Given: A minimal valid policy for add=true (merge mode)
+    When: _validate_bucket_policy_for_set is called
+    Then: No exception is raised
+    """
+    from GCP import _validate_bucket_policy_for_set
+
+    policy = {"bindings": [{"role": "roles/storage.objectViewer", "members": ["user:alice@example.com"]}]}
+
+    # Should not raise
+    _validate_bucket_policy_for_set(policy, add_mode=True)
```

Comment:
Maybe use `pytest.raises` and verify there are no errors. Otherwise, the test will fail due to the error instead of the test itself.